packages/api/tests/lambdas/test-db-indexer.js

```diff
@@ -394,3 +412,47 @@ test.serial('The db-indexer does not throw an exception when execution fails', a
     })(() => handler({ Records: [insertRecord] }))
   );
 });
+
+test.serial('create, update and delete a reconciliationReport in DynamoDB and ES', async (t) => {
```

Comment:
This test should be for the model and can be split up into a few tests. The test in this file should be a super simple test of just the dbIndexer functionality.

Reply:
Removed the test from dbIndexer.  It's not necessary to add to the model test, all basic functions from model base.